### ISSUE TITLE
Normalize underscores in field titles for downloaded data

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#743]
# TL;DR
When you download the child data from a study there are two underscores in the child id column name (child _ _ hashed _ id) but in the frame data sheet it is just one underscore (child _ hashed _ id).

# Narrative
an idea for a small win in between bigger feature development: when you download the child data from a study there are two underscores in the child id column name (child _ _ hashed _ id) but in the frame data sheet it is just one underscore (child _ hashed _ id) - I know this is a tiny thing but it's something you have to account for when merging the 2 frames in R to look at analyses by age.

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Unsure if this request is for keeping the two underscores or if this is a limitation of the frame data sheet.  A bit a research might be needed to understand the scope and work required.  (My lack of R knowledge might be showing a bit here.)

**Comment by okaycj** [comment]
As long as the researchers don't find this annoying, we can let it go.  Closing.
